### DISCUSSION TITLE
Load Whisper in Triton Inference Server

### DISCUSSION THREAD

**Discussion by Astenseke** [triton-inference-server/server#6244] [General]
Has anyone tried this successfully?

**Comment by outtanames** [comment]
Not sure if Triton supports this out of the box, but there's a tutorial on https://github.com/autonomi-ai/nos for running Whisper with a REST API https://docs.nos.run/#audio-transcription-whisper-as-a-service.

**Comment by saitejamalyala** [comment]
https://github.com/triton-inference-server/server/issues/7847

Has a example using python backend (there are better ways to do it as well). 

@Astenseke
